blog-post.html

```diff
@@ -1,12 +1,162 @@
+      start messing with your junk? Ann, you need to decide whether you want a man or a boy. I know how I'd answer.
+      She's a girl, I need to teach her how to be a woman. Within her lies a queen. Let me out that queen. I may have
+      committed some light treason.
+
+      You're a good guy, mon frere. That means brother in French. I don't know how I know that. I took four years of
+      Spanish. I'm foolish and I'm funny and I'm needy. Am I needy? Are you sure I'm not needy? 'Cause I feel needy
+      sometimes. No! These are just strippers! Look how hot they are! We'll have to find something to do so that people
+      can look at you without wanting to kill themselves. It's as Ann as the nose on Plain's face. That was Tom Cruise,
+      the actor. They said he was some kind of scientist. GENE!! [screams]
+    </p>
+    </p>
+    <p>Let's see some bananas and nuts! Her lawyers are claiming the seal is worth $250,000. And that's not even
+      including
+      Buster's Swatch. But I did finally get into Dad's pants. Although I had to have the crotch taken in a little
+      bit.
+      The worst that could happen is that I could spill coffee all over this $3,000 suit. COME ON. You want your belt
+      to
+      buckle, not your chair. I'm gonna go get sexy. </p>
+
+    <p>I'm tired of trying to find happiness through lies and self-medicating. If you need me, I'll be at the bar.
+      It's
+      a jetpack, Michael. What could possibly go wrong?</p>
+  </article>
+
+
+
+  <!-- footer with submit fields -->
+
+  <!-- contact section - right hand side of footer-->
+
+  <footer class="item footer">
+    <h3 id='foottitle'>Questions or Comments? Get In Touch:</h3>
+    <br>
+
+
+    <form id='name'>
+      <label for='fname'></label>
+      <input type='text' value='Name'>
+    </form>
+
+    <form id='email'>
+      <label for='lname'></label>
+      <input type='text' value='Email'>
+    </form>
+
+    <form id='message'><br>
+      <label for='message'>Message:</label><br>
+      <input inputmode="text" id='message' name='message'><br>
+      <input type='submit' value="Submit">
+    </form>
+
+    <p id='contacttitle'>
+      We got a fakebook, the tweeter, or just call the totally real number!
+    </p>
+
+    <item id='address'>
+      <img src=images/home.png class='icon'>123 Fake Street
+    </item>
+    <item id='phone'>
+      <img src=images/phone.png class='icon'>(000) 000-0000
+    </item>
+    <item id='mail'>
+      <img src=images/email.png class='icon'>info@nunyabiz.kom
+    </item>
+    <item id='twit'>
+      <img src=images/twitter.png class='icon'>@tweeter
+    </item>
+    <item id='insta'>
+      <img src=images/instagram.png class='icon'>instasham.com
+    </item>
+    <item id='drib'>
+      <img src=images/dribbble.png class='icon'>dribble.com
+    </item>
+    <item id='face'>
+      <img src=images/facebook.png class='icon'>fakebook.com
+    </item>
+
+  </footer>

```

Comment:
This is a great start to your footer, we can adjust this to make it closer to the final version by adjusting the sizes of your grid boxes. This goes for all of your footers.